qa/tasks/s3tests.py

```diff
@@ -229,6 +229,9 @@ def run_tests(ctx, config):
     assert isinstance(config, dict)
     testdir = teuthology.get_testdir(ctx)
     attrs = ["!fails_on_rgw", "!lifecycle"]
+    # beast parser is strict about unreadable headers
+    if ctx.rgw.frontend == 'beast':
+        attrs.append("!fails_strict_rfc2616")
```

Comment:
Just a heads up, Beast tries to adhere strictly to rfc7230, which replaces rfc2616.